src/components/CompareTable/CompareCompetencesTableRow.tsx

```diff
@@ -0,0 +1,35 @@
+import { components } from '@/api/schema';
+import { capitalize, getLocalizedText } from '@/utils';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+export interface CompareCompetencesTableRowData {
+  uri: string;
+  nimi: components['schemas']['LokalisoituTeksti'];
+  kuvaus: components['schemas']['LokalisoituTeksti'];
+  profiili?: boolean;
+}
+
+interface CompareCompetencesTableRowProps {
+  row: CompareCompetencesTableRowData;
+  className?: string;
+}
+
+export const CompareCompetencesTableRow = ({ row, className }: CompareCompetencesTableRowProps) => {
+  const { t } = useTranslation();
+  const title = React.useMemo(() => getLocalizedText(row.kuvaus), [row.kuvaus]);
+
+  return (
+    <tr className={className}>
+      <td className="w-full pl-5 pr-7 py-3 text-heading-4 hyphens-auto" title={title}>
+        {capitalize(getLocalizedText(row.nimi))}
```

Comment:
The first letter can be capitalized also with just the CSS if wanted and with TailwindCSS it seems to work like this: `first-letter:uppercase`